tsconfig.json

```diff
@@ -3,14 +3,17 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist/out-tsc",
+    "forceConsistentCasingInFileNames": true,
```

Comment:
Are these options new additions to ts? We'll just need to pay attention to these changes in our downstream.

Reply:
These aren't new additions to ts, they are just newly recommended to be used by Angular.  As part of the update process, I generated a new Angular 10 project and compared the differences between our configuration and the Angular defaults.  These options can always be disabled downstream if necessary.